x-pack/plugins/graph/public/app.js

```diff
@@ -715,9 +719,14 @@ app.controller('graphuiPlugin', function ($scope, $route, $interval, $http, kbnU
       .on('zoom', redraw));
 
 
+  const managementUrl = chrome.getNavLinkById('kibana:management').url;
+  const url = `${managementUrl}/kibana/indices`;
 
   if ($scope.indices.length === 0) {
-    notify.warning('Oops, no data sources. First head over to Kibana settings and define a choice of index pattern');
+    toastNotifications.addWarning({
+      title: 'No data sources',
+      text: <p>Go to <a href={url}>Management &gt; Index Patterns</a> and create an index pattern</p>,
+    });
   }
 
```

Comment:
No data source

Go to **Management > Index Patterns** and create an index pattern. 

Notes:
sources -> source
add period after pattern